test/data/valid_openapi/readOnly.json

```diff
@@ -0,0 +1,80 @@
+{
+	"openapi": "3.0.0",
+	"info": {
+		"version": "1.0.0",
+		"title": "Swagger Petstore"
+	},
+	"servers": [
+		{
+			"url": "http://petstore.swagger.io/v1"
+		}
+	],
+	"paths": {
+		"/pets": {
+			"get": {
+				"responses": {
+					"200": {
+						"description": "Successfull",
+						"content": {
+							"application/json": {
+								"schema": {
+									"type": "array",
```

Comment:
can we use this below defined schema as a ref in both ? This would give more logical explanations as to why the properties readOnly and writeOnly are used